crm_won_reason/wizard/crm_lead_won.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields, models
+
+
+class CrmLeadReason(models.TransientModel):
+    _name = "crm.lead.won"
+    _description = "Get Won Reason"
+
+    won_reason_id = fields.Many2one(
+        "crm.lost.reason", "Won Reason", domain="[('reason_type','in',[False,'won'])]"
+    )
+
+    def action_win_reason_apply(self):
+        leads = self.env["crm.lead"].browse(self.env.context.get("active_ids"))
+        return leads.action_set_won(won_reason_id=self.won_reason_id.id)
```

Comment:
It seems this method isn't tested.
Nitpicking here but you can easily achieve 100% coverage by slightly changing tests

Reply:
that's right

Reply:
@ivantodorovich  I changed but still not close to 100% percent coverage, i need it for customer for today so please approve if code LG to you.

Reply:
Oh, I've already approved on my first review 😅
These are just nitpicking suggestions, IMO it's ready.
Thanks for this module